### DISCUSSION TITLE
How can you tell where the song is being played from?

### DISCUSSION THREAD

**Discussion by vkkodali** [KRTirtho/spotube#1428] [Q&A] [answered]
I have Jiosaavn selected as the Audio Source but I also logged in to my Spotify account. Is there any way to tell which source is being used to play the song?

**Comment by KRTirtho** [comment, marked as answer]
There's a icon button in the player named Alternative Track sources 
![image](https://github.com/KRTirtho/spotube/assets/61944859/513e6eef-c0d9-4c3d-8991-0a34db5d3630)

**Comment by vkkodali** [comment]
Thanks, but that icon only tells me what _other_ sources are available for the track. How can I tell what the _current_ source of the track is?

**Reply by KRTirtho** [reply]
The selected source is the current sourcer

**Reply by vkkodali** [reply]
Got it! Thank you
